presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -78,7 +82,7 @@ Array Functions
 .. function:: array_has_duplicates(array(T)) -> boolean
 
     Returns a boolean: whether ``array`` has any elements that occur more than once.
-    Throws an exception if any of the elements are rows or arrays that contain nulls. 
+    Throws an exception if any of the elements are rows or arrays that contain nulls.
 
     SELECT array_has_duplicates(ARRAY[1, 2, null, 1, null, 3]) -- true
```

Comment:
Please fix the formatting for the code example - see screenshot of local doc build. 
<img width="803" alt="Screenshot 2024-09-04 at 1 06 55 PM" src="https://github.com/user-attachments/assets/5c34e125-d370-49d5-9dcd-ca95abecdb7d">

Reply:
Should we do this in a differ PR as this PR is for ARRAY_CONTAINS_ALL?

Reply:
You're right, ignore this. I'll deal with it in a separate PR.

Reply:
#23583